.gitignore

```diff
@@ -3,3 +3,4 @@
 .php_cs.cache
 composer.lock
 phpunit.xml
+/.phpunit.result.cache
```

Comment:
Shall we add this to dev kit?

Reply:
The whole gitignore?

Reply:
Hmm yes I think we could manage them mostly the same. Let's see afterwards if we have some special cases and add them via an `{%if ...}` like for other config files, WDYT? Open to create a PR?

Reply:
Will do it

Reply:
So we would list other config files in `projects.yaml` or what?